index.html

```diff
@@ -154,10 +154,10 @@
     <script src="/vendor/scripts/webdb.js"></script>
     <script src="/vendor/scripts/page.js"></script>
     <!-- Model files: -->
+    <script src="/scripts/views/articleView.js"></script>
```

Comment:
This has to be placed beneath the script tag for article.js because the articleView files references the Article object exported from article.js (article.js needs to have loaded first).